<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mantCategoria</class>
 <widget class="QMainWindow" name="mantCategoria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingresar Medio de Pago</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCodigoMedioPago">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CÃ³digo             :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMedioPago">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Medio De Pago :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMedioPago">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>INGRESAR UN NUEVO MEDIO DE PAGO</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtIdMedioPago">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
